k8s/cmd/nsmdp/nsmdp.go

```diff
@@ -41,10 +42,18 @@ const (
 	resourceName = "networkservicemesh.io/socket"
 	// ServerSock defines the name of NSM client socket
 	ServerSock = "networkservicemesh.io.sock"
+
+	// A number of devices we have in buffer for use.
+	DeviceBuffer = 30
```

Comment:
Is this an arbitrary number? What if we need 31?

Reply:
On every allocation, it adds +1 device. It is buffer to hold +30 available ones.